.github/workflows/docker-release.yml

```
@@ -10,7 +10,7 @@ jobs:
       matrix:
         image: [
           { name: "divviup_api", rust_features: ["default"] },
```

This one should be updated too
```suggestion
          { name: "divviup_api", rust-features: ["default"] },
```